src/components/Onboarding/Share/SharePreview.js

```diff
@@ -156,15 +156,15 @@ export const SharePreview = ({
           el => el.channelIdentifier === shareChannel?.channelIdentifier
         )?.name
       ];
-    const title = `Bring das Grundeinkommen mit mir an den Staat! Melde dich dafür bei der Expedition Grundeinkommen an. Ich bin schon in ${municipality.name} dabei :)`;
+    const title = `Bring das Grundeinkommen mit mir an den Staat! Melde dich dafür bei der Expedition Grundeinkommen an.`;
```

Comment:
Passt der Satz noch in Bezug auf das Crowdfunding?

Reply:
Ich hab es so gelassen, damit es für beide Fälle passt. Bin aber offen für weitere Vorschläge! :)

Reply:
Schwierig, mir ist auch nicht wirklich was eingefallen. Wahrscheinlich ist es ganz gut so. :)